extras/parse_travis_log.py

```diff
@@ -899,7 +900,6 @@ def build_return_code(status_vera_init,
        (status_make_install) and \
        (status_tests) and \
        (number_of_msgs_in_summary(summary_vera) == 0) and \
-       (number_of_msgs_in_summary(summary_cppcheck) == 0) and \
```

Comment:
@heplesser, @jougs 

Fundamentally, I vote for following clean coding guidelines, and having a styleguide is definitely a must. Since you spent lot more time reading code than writing, we want the reading of code to be easy. Therefore, we should be careful in going too far and limit ourselves where it is not needed. For example: The 79 characters per line limitation makes it more difficult to produce good readable code. This discussion is very very old. My personal experience is, that sooner or later people switch to 100 ... 120.    

Some of the backslashes are a result of that restriction. I need to change that at several places in the code. 
Question: Is there a way to force PEP8 to complain about that ? If so, we should add that to the code analysis.

Concerning the looong function name: It seems that I became a victim of my own naming convention here. A shorter name would still be descriptive enough. I will replace the prefix "number_of_msgs_" by something shorter and also have a look a the names again.